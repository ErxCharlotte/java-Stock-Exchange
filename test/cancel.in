ADD trader1 100.00
ADD trader2 100.00
IMPORT trader1 ABC 100.00
IMPORT trader1 CDE 500.00
SELL trader1 ABC 100 5
INVENTORY trader1
CANCEL SELL 1000
CANCEL SELL 0000
INVENTORY trader1
BUY trader1 ABC 100 5
CANCEL BUY 1000
CANCEL BUY 0001
BOOK BUY
EXIT