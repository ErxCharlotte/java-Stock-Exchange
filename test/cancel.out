$ Success.
$ Success.
$ Trader now has 100.00 units of ABC.
$ Trader now has 500.00 units of CDE.
$ No trades could be made, order added to sell book.
$ CDE
$ No such order in sell book.
$ Order successfully cancelled.
$ ABC
CDE
$ No trades could be made, order added to buy book.
$ No such order in buy book.
$ Order successfully cancelled.
$ The buy book is empty.
$ Have a nice day.